### PR TITLE
Update microsoft-defender-endpoint-linux.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note of support for previous versions.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6943

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ If you experience any installation failures, refer to [Troubleshooting installat
        - For 6.9: 2.6.32-696.* 
        - For 6.10: 2.6.32.754.2.1.el6.x86_64 to 2.6.32-754.41.2:
 
+ > [!NOTE]
+ > After a new package version is released, support for the previous two versions is reduced to technical support only. Versions older than that are listed in this section, are provided for technical upgrade support only.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md]:
```suggestion
 > After a new package version is released, support for the previous two versions is reduced to technical support only. Versions older than that that are listed in this section are provided for technical upgrade support only.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Pleas merge this PR. Thanks

**Pull request merged**